Make test is broken

From master
```
make test
PATH=/usr/local/openresty/nginx/sbin:$PATH prove -I../test-nginx/lib -r t
t/sanity.t .. 44/72 # Looks like you planned 72 tests but ran 74.
t/sanity.t .. Dubious, test returned 255 (wstat 65280, 0xff00)
All 72 subtests passed 

Test Summary Report
-------------------
t/sanity.t (Wstat: 65280 Tests: 74 Failed: 2)
  Failed tests:  73-74
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 72 tests but ran 74.
Files=1, Tests=74,  2 wallclock secs ( 0.04 usr  0.00 sys +  0.38 cusr  0.13 csys =  0.55 CPU)
Result: FAIL
make: *** [Makefile:17: test] Error 1
```
Using git bisect
```a2cb7798133d891510bd37340390314644083da9 is the first bad commit```


a2cb7798133d891510bd37340390314644083da9 is the commit adding a test but not updating `plan tests`

--- a/t/sanity.t
+++ b/t/sanity.t
@@ -5,7 +5,7 @@ use Cwd qw(cwd);
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 3 + 6);
+plan tests => repeat_each() * (blocks() * 3 + 7);
 
 my $pwd = cwd();
 

